DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/PersonalInformation.cshtml

```diff
@@ -0,0 +1,70 @@
+@using DigitalLearningSolutions.Web.ViewModels.Register
+@model PersonalInformationViewModel
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Register Delegate" : "Register Delegate";

```

Comment:
As this is part of the Tracking System, I think we'll also want to define on these ViewData variables on these two new pages:
  ViewData["Application"] = "Tracking System";
  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
  ViewData["HeaderPathName"] = "Tracking System";
like the other Tracking System pages so that the layout can function properly.